docs/MULTIREPO.md

```diff
@@ -0,0 +1,91 @@
+# This doc describes a workflow for using multiple repositories in a development flow.
+
+We will have 2 repositories set up: App and Overlay.
+
+The App repository will contain everything to build the app on a developers machine.
+The App repository will contain an empty theme directory. When looking for an 
+The Overlay repository will contain:
+  - A submodule, which is the App repository
+  - An overlay directory
+  - Build scripts that will ensure the build uses files from the overlay, if they exist and falls back to defaults otherwise
+
+The overlay directory can only contain white-listed paths; this prevents the Overlay repository from accidentally adding features. 
+Furthermore the overlay directory can only contain files that are present in the App repository; this prevents situations where the App repository doesn't build without the Overlay repository.
```

Comment:
Even though this would add a bit of overhead to find a viable dummy asset, this is a **must** imo. Maybe we can do a little bit of up-front work finding a library of usable dummy assets